typedoc.json

```diff
@@ -3,7 +3,7 @@
   "entryPoints": ["./src/core/index.ts", "./src/react/index.ts"],
   "out": "typedoc/generated",
   "readme": "typedoc/landing-page.md",
-  "treatWarningsAsErrors": true,
+  "treatWarningsAsErrors": false,
```

Comment:
Why have we changed this? Treating warnings as errors helps us identify things that are missing documentation.

Reply:
Ahh yes, this was done during testing while none of the typedocs worked, I missed setting this one back, I'll correct that now.